Source/AliveLibAO/MainMenu.cpp

```diff
@@ -3269,12 +3269,36 @@ void Menu::ButtonRemap_Update_47F6F0()
         field_1E0_selected_index--;
         SFX_Play_43AE60(SoundEffect::MenuNavigation_61, 45, 400, 0); // TODO: Input constants
         bWaitingForRemapInput_9F2DE8 = 1;
+
+#if ORIGINAL_PS1_BEHAVIOR
+        if (!Input_JoyStickEnabled())
+        {
+            if (field_1E0_selected_index < 0)
+            {
+                field_1E0_selected_index = (int)RemapOption::eCrouch_6;
```

Comment:
seems like field_1E0_selected_index needs changing to be of type RemapOption? That will ditch these casts, if we do need a cast it should be a C++ cast (static_cast<T>(a)) in this case.

Reply:
field_1E0_selected_index is used for ALL main menu selections, not just the controls remap page.
namespace enums ditch the cast and won't clutter the namespace. What are your thoughts on them?

Reply:
What about using a union with an enum class and a raw type inside of it like the Abe.hpp?